src/lifecycle/properties-ready.js

```diff
@@ -0,0 +1,6 @@
+export default function propertiesApply (elem, props) {
+  Object.keys(props).forEach(function (name) {
+    let prop = props[name];
+    prop.set && prop.set.call(elem, elem[name]);
```

Comment:
`prop.set` will always be defined, right? That is, this isn't the `set` *option*.